jbt-it/neomdb
# #255 add new test database configuration

Setup test database configuration and api tests.
How to execute tests:
- Download newest .env file fro teams
- Create mdb_test in mysql docker
- Create new test user (credentials are saved in .env)
- Install VSC extension: Jest
- Navigate to the test file or to the Testing Tab (see image)
- Execute tests by clicking on the arrows (see image 2)

![image](https://github.com/jbt-it/neomdb/assets/45009167/953f3c5a-803a-4ae4-b2d9-c6218ebb235e)

![image](https://github.com/jbt-it/neomdb/assets/45009167/e08abe5d-ee08-4797-95cc-592b2441c148)
